src/odemis/driver/piezomotor.py

```diff
@@ -650,13 +712,26 @@ def _sendCommand(self, cmd):
         cmd += EOL
         with self._ser_access:
             logging.debug("Sending command %s", to_str_escape(cmd))
-            self._serial.write(cmd)
+            try:
+                self._serial.write(cmd)
+            # TODO: what kind of exception is raised? Needs to be more specific.
```

Comment:
Indeed... please pick an exception.

Reply:
Problem is, I don't know which exception is raised. IOError is not the right one, I think it's something very specific, we need to test this with someone in the lab.

Reply:
The tmcm driver uses `IOError`, and it works fine. So I'd go for that too. It might be possible to be even more specific, but I don't think is really necessary.

Reply:
I tried IOError, it didn't work, but I didn't have time to check what it was in the end.